src/Mysql/Select.php

```diff
@@ -91,6 +92,19 @@ public function addValue($tableId, $column)
         return self::insert($this->columns, $name);
     }
 
+    public function addCount()
```

Comment:
This is a very specific function. We won't be able to keep up with the infinite variety of MySQL expressions if we hand-code a method for each one.

We could add a new MySQL expression for the "COUNT" function. Then we'd be able to use the "add" method to add any expression--whether it is a column or a function (or anything else, frankly).